app/experimenter/nimbus-ui/src/components/PageRequestReview/DraftStatusOperations.tsx

```diff
@@ -118,6 +134,40 @@ export const DraftStatusOperations = ({
         />
       );
 
+    /* istanbul ignore next until EXP-1055 & EXP-1062 done */
+    case DraftStatusOperationsState.LaunchRejected:
+      return (
+        <Alert variant="warning">
+          <div className="text-body">
+            <p>
+              Your experiment was <strong>Rejected</strong> due to:
+            </p>
+            <p className="mb-0">
+              {rejectFeedback!.rejectedByUser} on {rejectFeedback!.rejectDate}:
+            </p>
+            <p className="bg-white rounded border p-3">
+              {rejectFeedback!.rejectReason}
+            </p>
+          </div>
+        </Alert>
+      );
+
+    /* istanbul ignore next until EXP-1055 & EXP-1062 done */
+    case DraftStatusOperationsState.LaunchRejectedByAnotherUser:
```

Comment:
I need to run this one by Ana. I'm not sure what we want to display if the experiment has been rejected but the current user 1) does not have reviewer role and does not own the experiment, and 2) if user has a reviewer role but didn't do the rejecting.